File: 1-Semestre/Desarrollo de algoritmos/Actividades/Arreglos (Semana 13)/Ejercicio 12.py
'''
Hacer un programa en python que lea n números y los guarde en un arreglo A, ordenar los números
de menor a mayor y guardarlos ordenados en un arreglo B, imprimir ambos arreglos.
'''

n = 5
A = []

for i in range(n):
    dato = int(input(f"Ingrese un número (nº{i+1}): "))
    A.append(dato)

B = sorted(A) # Ordena la lista, en este caso de menor a mayor. (Se usa sorted y no A.sort() porque sino devuelve None al asignarlo a otra variable)
print(f"Arreglo A (original): {A}")
print(f"Arreglo B (ordenado): {B}")

File: 2-Semestre/Taller de Programación/Personal/Arreglos/Ejercicio 12.cs
/*
Hacer un programa en python que lea n números y los guarde en un arreglo A, ordenar los números
de menor a mayor y guardarlos ordenados en un arreglo B, imprimir ambos arreglos.
*/

using System;
using System.Collections.Generic;

namespace Ejercicio_12
{
    public class Program
    {
        public static void Main(string[] args)
        {
            List<int> A = new List<int>();

            int n = 4;

            for (int i = 1; i <= n; i++)
            {
                Console.Write($"Ingrese un número #{i}: ");
                int dato = Convert.ToInt32(Console.ReadLine());

                A.Add(dato);
            }

			Console.WriteLine($"\nArreglo A (Original): {string.Join(", ", A)}");
            A.Sort();
            Console.WriteLine($"\nArreglo B (Ordenado): {string.Join(", ", A)}");
        }
    }
}